### PR TITLE
[all] (convert Visual Studio project to TypeScript) Fix issues

### PR DESCRIPTION
**Pull request by kbrandl**
This PR fixes problems with the [Convert an Office Add-in project in Visual Studio to TypeScript](https://docs.microsoft.com/office/dev/add-ins/develop/convert-javascript-to-typescript) article that were preventing the guidance from working in Visual Studio 2019.

### CODE REVIEW

**Comment by kbrandl** [comment]
@AlexJerabek here's the PR that we discussed earlier; thanks in advance for your feedback!

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fe81df1](https://github.com/OfficeDev/office-js-docs-pr/commits/fe81df14004a2cd538a2beb57f59e1f34b8c072d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-vs-2019-issue/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910292002473880-1399%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=5tslAdpLCg22qkf4QoGjNnZyOU9JK0C%2Baaec9wfQ6MM%3D&st=2019-10-29T19%3A58%3A51Z&se=2019-11-29T20%3A03%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/convert-javascript-to-typescript.md [hunk 1]
```diff
@@ -42,22 +42,29 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
...
+7. In **Home.ts**, find the line `if(!Office.context.requirements.isSetSupported('ExcelApi', '1.1') {` and replace it with the following:
 
     ```TypeScript
-    declare var fabric: any;
+    if(!Office.context.requirements.isSetSupported('ExcelApi', 1.1)) {
     ```
 
-6. In the **Home.ts** file, find the line `Office.initialize = function (reason) {` and add a line immediately after it to polyfill the global `window.Promise`, as shown here:
+    > [!NOTE]
+    > Currently, for the project to compile successfully after it's converted to TypeScript, you must specify the requirement set number as a numeric value as shown in the previous code snippet. Unfortunately this means you'll be unable to use `isSetSupported` to test for requirement set `1.10` support, as the numeric value `1.10` evaluates to `1.1` at runtime. This article will be updated when this issue is resolved.
```
Comment (AlexJerabek) [inline docs/develop/convert-javascript-to-typescript.md]: Should we specify this issue is due to a versioning problem? If so, we could also call out the out-of-date types in this section.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0996519](https://github.com/OfficeDev/office-js-docs-pr/commits/0996519f539452c840304412871371497867dca6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-vs-2019-issue/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910292019253551-1399%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=QwcPuy4EBqnpAhkgskUiv4f%2BnjAXexNRIVsdXTAsyzI%3D&st=2019-10-29T20%3A15%3A24Z&se=2019-11-29T20%3A20%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
Good feedback, thanks @AlexJerabek! I've updated the note per your suggestion.

**Pull request merged**